cloud_utils/scheduler/kubernetes_connector.py

```diff
@@ -197,6 +198,13 @@ def _make_job_spec(
     },
 )
 
+_DEFAULT_JOB_ANNOTATIONS = {
```

Comment:
not sure this belongs here... can we move it to the relevant repo (this is open source after all)